web/app/components/header/toolbar.hbs

```diff
@@ -14,12 +14,13 @@
               @label="Type"
               @facets={{@facets.docType}}
               @disabled={{not @facets.docType}}
+              class=""
```

Comment:
Any reason for this empty class attribute? (more curious vs. strong feelings against it)